### ISSUE TITLE
Rename dist_analysis.py

### ISSUE THREAD

**Issue by tseemann** [TGAC/KAT#33]
As per recommendations in http://www.gigasciencejournal.com/content/2/1/15 I would suggest renaming

* `dist_analysis.py` => `kat-dist_analysis`
* `jellyfishk` => `kat-jellyfish`

**Comment by maplesond** [comment]
Ok, I think that naming convention is clearer.  I'll make these changes in V2.1.

**Comment by maplesond** [comment]
I can't use hypens in compiled executable as this appears to break automakes rules:

```
Makefile.am:40: error: bad characters in variable name 'bin_kat-jellyfish_SOURCES'
```
 
So jellyfish will be renamed to `katjellyfish` and I'll retain the `kat-` prefix for the scripts.

**Comment by maplesond** [comment]
Correction... to maintain consistency I will rename jellyfish `kat_jellyfish` and the scripts will start with `kat_` prefix.

**Comment by maplesond** [comment]
Fixed in head of develop branch, will be in V2.1

**Comment by maplesond** [comment]
Should be fixed in the latest release: https://github.com/TGAC/KAT/releases/tag/Release-2.1.0.  Please reopen if there are any issues.
